x-pack/plugins/security_solution/common/api/endpoint/actions/status/status.schema.yaml

```diff
@@ -25,5 +25,44 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '../../model/schema/common.schema.yaml#/components/schemas/SuccessResponse'
+                $ref: '#/components/schemas/ActionStatusSuccessResponse'
+
+components:
+  schemas:
+    PendingActionsSchema:
+      type: object
+      properties:
+          'isolate':
```

Comment:
@maximpn Is there a way to reuse the common [`Command`](https://github.com/elastic/kibana/blob/6d46edc275ba670a885acfc40e145a99b7abbf78/x-pack/plugins/security_solution/common/api/endpoint/model/schema/common.schema.yaml#L46) schema property here instead of writing it all down.

Reply:
I'm afraid it's impossible to reuse `enum` schema to restrict allowed keys. There are two ways to proceed depending on what keys should be allowed

- **Only certain keys are allowed.** Define a value's schema and reuse it via `$ref` for each property definition. (In this particular case values have type `integer` which is trivial and shared schema isn't strictly necessary though it'd improve readability by highlighting that each property has exactly the same schema).
- **Arbitrary keys must have the same schema (TS equivalent is `Record<string, MySchema>`)**. `additionalProperties` allow to make this trick as [ObjectQueries](https://github.com/elastic/kibana/blob/main/x-pack/plugins/osquery/common/api/model/schema/common_attributes.schema.yaml#L210-L213) definition does

```yaml
ObjectQueries:
  type: object
  additionalProperties:
    $ref: '#/components/schemas/ObjectQueriesItem'
```

Reply:
@maximpn Thanks for the clarification.